templates/advantage/index.html

```diff
@@ -307,31 +296,28 @@ <h2>Your paid subscriptions</h2>
-                              </p>
-                            </div>
-                          </div>
-                          <div class="col-12 u-align--center">
-                            To activate FIPS: <code>sudo ua enable fips</code>
-                          </div>
-                        </div>
-                      </td>
-                      <td id="expanded-row-cc-eal-{{ outer_loop.index }}-{{ loop.index }}" class="p-table-expanding__panel" aria-hidden="true">
-                        <div class="row">
-                          <div class="col-12">
-                            <div class="p-notification--caution">
-                              <p class="p-notification__response">
-                                Older versions of software are used for CC-EAL-certified packages. Use them only if your organisation requires it.
-                              </p>
-                            </div>
-                          </div>
-                          <div class="col-12 u-align--center">
-                            To activate CC-EAL: <code>sudo ua enable cc-eal</code>
+                          <div class="col-8 col-start-small-1 col-start-medium-2 col-start-large-4">
+                            <p class="u-no-margin--bottom"><strong>FIPS 140-2</strong> Level 1 certified packages use older software versions.</p>
+                            <p>Use them only if your organisation requires it.</p>
+                            <ul class="p-list" style="padding-left: 2rem;">
+                              <li>Available on: Ubuntu 16.04 LTS, 18.04 LTS</li>
+                              <li>To activate FIPS: <a class="p-link--external" href="https://docs.ubuntu.com/security-certs/en/fips">FIPS setup instructions</a></li>
+                            </ul>
+
+                            <p class="u-no-margin--bottom"><strong>CC-EAL2</strong> certified packages use older software versions.</p>
+                            <p>Use them only if your organisation requires it.</p>
+                            <ul class="p-list" style="padding-left: 2rem;">
+                              <li>Available on: Ubuntu 16.04 LTS</li>
+                              <li>To activate CC-EAL2: <a class="p-link--external" href="https://docs.ubuntu.com/security-certs/en/cc-16/">CC-EAL2 setup instructions</a></li>
+                            </ul>
+                            
+                            <p><strong>CIS tools</strong> test compliance with the CIS security benchmark.</p>
+                            <ul class="p-list" style="padding-left: 2rem;">
+                              <li>Available on: Ubuntu 16.04 LTS, 18.04 LTS</li>
+                              <li>To use the tools: <a class="p-link--external" href="https://docs.ubuntu.com/security-certs/en/cis-18-16.html">CIS setup instructions</a></li>
+                            </ul>
```

Comment:
Alignment was looking a little weird - but if we do this (based off what's in the "Infra" tab), it looks neater:
```suggestion
                          <div class="col-12 u-align--center">
                            <strong>FIPS 140-2</strong> Level 1 certified packages use older software versions.<br />
                            Use them only if your organisation requires it.<br /><br />
                            
                            Available on: Ubuntu 16.04 LTS, 18.04 LTS<br />
                            To activate FIPS: <a class="p-link--external" href="https://docs.ubuntu.com/security-certs/en/fips">FIPS setup instructions</a><br /><br />
                            
                            <strong>CC-EAL2</strong> certified packages use older software versions<br />
                            Use them only if your organisation requires it.<br /><br />
                            
                            Available on: Ubuntu 16.04 LTS<br />
                            To activate CC-EAL2: <a class="p-link--external" href="https://docs.ubuntu.com/security-certs/en/cc-16/">CC-EAL2 setup instructions</a><br /><br />
                            
                            <strong>CIS tools</strong> test compliance with the CIS security benchmark.<br /><br />

                            Available on: Ubuntu 16.04 LTS, 18.04 LTS<br />
                            To use the tools: <a class="p-link--external" href="https://docs.ubuntu.com/security-certs/en/cis-18-16.html">CIS setup instructions</a>
```